frontend/packages/dev-console/src/components/helm/HelmChartSummary.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { HelmRelease } from './helm-types';
+import { Timestamp } from '@console/internal/components/utils';
+
+interface HelmChartSummaryProps {
+  helmRelease: HelmRelease;
+}
+
+const HelmChartSummary: React.FC<HelmChartSummaryProps> = ({ helmRelease }) => (
+  <dl className="co-m-pane__details">
```

Comment:
nit : can we avoid  repeating `helmRelease.chart.metadata.*` can destructure it or just assign 

```
const {
    chart: {metadata: {name, version: chartVersion},
    version, 
    info: {last_deployed}
  } = helmRelease;
```

Reply:
Updated.